cluster-autoscaler/utils/errors/errors.go

```diff
@@ -71,7 +71,7 @@ func ToAutoscalerError(defaultType AutoscalerErrorType, err error) AutoscalerErr
 	if e, ok := err.(AutoscalerError); ok {
```

Comment:
Shouldn't we return nil from ToAutoscalerError if err is nil? It seems weird that we construct an AutoscalerError when there is actually no error.

Reply:
It's still an error message, and even err is nill, the error type (defaultType) isn't. So we should still return an error here.